backend/api/test_views.py

```diff
@@ -1190,3 +1190,68 @@ def test_detail_match_unsubmitted_returns_404(self):
                     response = self.client.get(url)
 
                     self.assertEqual(response.status_code, 404)
+
+
+class SchemaViewTests(TestCase):
+    def setUp(self):
+        self.fiscal_years = [
+            "2023",
+        ]
+
+        self.schema_types = [
+            "metadata",
+        ]
+
+        self.user = baker.make(User)
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.user)
+
+    def path(self, fiscal_year, type):
+        """Convenience method to get the path for a particular year and schema type)"""
+        return reverse("schemas", kwargs={"fiscal_year": fiscal_year, "type": type})
+
+    def test_authentication_required(self):
```

Comment:
Hmm, _is_ authentication required to get schemas? Should they be publicly available?

Reply:
Good question. Here's how I'm thinking about it - as things stand, authentication is (at least indirectly) required here, because you need to login in order to access the audit creation/submission steps. And more generally, I tend to lean toward a `deny by default, permit by exception` approach, where there need to be specific reasons to *not* require auth, rather than the other way around.

That said, there could very well be good reasons for these to be public. Any thoughts on this @tadhg-ohiggins ?

Reply:
My thought is that we're opening up the API to the world (with auth, of course) to interact with the site, so the schemas should be available to the world so they know what they're doing. I think your approach is good, but this is one of those things that felt it could be open since it doesn't deal directly with any data.

Reply:
Fair point, and agreed (especially in this case, since the schemas will be publicly available in any case with this being an open source project). I'll let @tadhg-ohiggins weigh in in case there are any other considerations here, but otherwise I'm happy to remove auth here

Reply:
Removing auth from this endpoint - thanks for feedback @geekygirlsarah!